# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../lua-subprocess/destdir.patch
# Copyright (C) 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./Makefile.orig	2011-06-24 19:21:40.000000000 +0200
+++ ./Makefile	2011-06-24 19:23:28.000000000 +0200
@@ -31,7 +31,7 @@
 
 .PHONY: install
 install: subprocess.so
-	$(INSTALL) -m755 subprocess.so $(INSTALL_CMOD)/
+	$(INSTALL) -m755 subprocess.so $(DESTDIR)$(INSTALL_CMOD)/
 
 .PHONY: uninstall
 uninstall:
